CHANGELOG.md

```diff
@@ -489,7 +489,12 @@ Non-backwards compatible changes
   * `fold-*`
   * `fold-pull`
 
-* In `Data.Fin.Properties` the `i` argument to `opposite-suc` has been made explicit.
+* In `Data.Fin.Properties`:
+  + the `i` argument to `opposite-suc` has been made explicit;
+  + the proof that an injection from a type `A` into `Fin n` induces a
+    decision procedure for `_≡_` on `A` has been generalized to other
+    equivalences over `A` (i.e. to arbitrary setoids), and renamed
+    from `eq?` to the more descriptive `inj⇒≟` and `inj⇒decSetoid`.
```

Comment:
Sorry another niggly organisation point. The generalisation is backwards compatible so should go under `Other minor changes`, the deprecation of `eq?` should live under the `Deprecated names` section. Other than that all looks good.

Reply:
Fixed. I hope it's as it should be now. 😄

Reply:
Yes, thanks!